packages/react-native/Libraries/Types/CodegenTypes.d.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import {EventSubscription} from '../vendor/emitter/EventEmitter';
+import {NativeSyntheticEvent} from './CoreEventTypes';
+// Event types
+// We're not using the PaperName, it is only used to codegen view config settings
+export interface BubblingEventHandler<T, PaperName extends string | undefined> {
+  (event: NativeSyntheticEvent<T>): void | Promise<void>;
+}
+
+export interface DirectEventHandler<T, PaperName extends string | undefined> {
+  (event: NativeSyntheticEvent<T>): void | Promise<void>;
+}
+// Prop types
+export type Double = number;
+export type Float = number;
+export type Int32 = number;
+export type UnsafeObject = any; // Object is forbidden in strict mode
+export type UnsafeMixed = any;
```

Comment:
I don't know if this has already been discussed, but I think it's preferable to use `unknown` instead of `any` here. `unknown` will create an error and make sure the consumer verifies the type of the value before using it. `any` will not notify the user that the type is unsafe.

Consumers can cast the value as `any` in their own code if they'd prefer to not verify the type.